docs/source/preflight/cli-usage.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Running Preflight checks
+description: Learn how to run preflight checks from the CLI
+---
+
+
+In order to check that a cluster and/or host meets the application's requirements prior to attempting an install, you are able to run preflight checks in advance.  Whilst this is built in to some applications, it's possible to run these via the CLI.
+
+## Usage
+
+In order to run, `preflight` needs a spec to understand what information to collect, and what to do with that information.  That spec can be a file, or hosted at a URL, located in an oci registry, or provided as stdin.
+
+To use stdin, supply `-` as the argument.
+
+Example usage:
+
+```shell
+./preflight https://preflight.replicated.com
+kubectl preflight oci://my.oci.registry/image
+./preflight my-preflight-spec.yaml
+helm template mychart --values my-values.yaml | ./preflight -
+```
+
+As of v0.64.0, stdin can contain the following as valid input for specs:
```

Comment:
I would adjust this wording a bit. Technically all methods of input support both `kind:` variants

Maybe:

> As of v0.64.0, valid input for specs can be either of: